From bda7ebe7ca032677aef14b94d9700e634d3d4ef2 Mon Sep 17 00:00:00 2001
From: fgm <fgm@27985.no-reply.drupal.org>
Date: Sat, 5 May 2012 17:30:02 +0200
Subject: [PATCH] Issue #1548240 by fgm, GiorgosK: $this->original_value was not initialized

- views_handler_field::original_value was only set in some cases.
- fixed a comment typo.
---
 handlers/views_handler_field.inc |    9 ++++++++-
 1 files changed, 8 insertions(+), 1 deletions(-)

diff --git a/handlers/views_handler_field.inc b/handlers/views_handler_field.inc
index c5834b8..1883d13 100644
--- a/handlers/views_handler_field.inc
+++ b/handlers/views_handler_field.inc
@@ -13,7 +13,7 @@
  */
 
 /**
- * Indicator of the render_text() mehtod for rendering a single item.
+ * Indicator of the render_text() method for rendering a single item.
  * (If no render_item() is present).
  */
 define('VIEWS_HANDLER_RENDER_TEXT_PHASE_SINGLE_ITEM', 0);
@@ -47,6 +47,13 @@ class views_handler_field extends views_handler {
   var $aliases = array();
 
   /**
+   * The field value prior to any rewriting.
+   *
+   * @var mixed
+   */
+  var $original_value = NULL;
+
+  /**
    * @var array
    * Stores additional fields which get's added to the query.
    * The generated aliases are stored in $aliases.
-- 
1.7.4.1

